src/SOS/SOS.Hosting/SymbolServiceWrapper.cs

```diff
@@ -116,7 +116,7 @@ protected override void Destroy()
                 {
                     symbolCachePath = _symbolService.DefaultSymbolCache;
                 }
-                if (!_symbolService.AddSymbolServer(msdl, symweb, symbolServerPath, authToken, timeoutInMinutes))
+                if (!_symbolService.AddSymbolServer(msdl, symweb, symbolServerPath, authToken, timeoutInMinutes != 0 ? timeoutInMinutes : null))
```

Comment:
What if someone actually wants to never retry?

Reply:
This code will go away when I port the setsymbolserver command to managed code.  I didn't want to waste time adding the option to the C++ command and the symbol service wrapper pinvoke.

Reply:
As long as we can push from c# to the native symbol path on WinDBG, sounds good to me.